documentation/prometheus-mixin/alerts.libsonnet

```diff
@@ -154,6 +154,20 @@
               description: 'Prometheus %(prometheusName)s is dropping {{ printf "%%.4g" $value  }} samples/s with timestamps arriving out of order.' % $._config,
             },
           },
+          {
+            alert: 'PrometheusTargetExceededBodySizeLimit',
```

Comment:
Hmm, I don't think we need this alert (or if we do, we should also add one for all the other scrape errors we have separate counters for).

I would suggest to stay in line with what we have so far, which is relying on users looking at their `up` metric to see scrape failures. We can discuss separately if specific alerts make sense and if so, add them in a separate PR.

Fro now, I think you should just add the new metric `prometheus_target_scrapes_exceeded_body_size_limit_total` to the dashboard for "Scrape failures" (in dashboards.libsonnet, line 57 and following).

Reply:
Yes, you are right, I was mislead because we have an alert for the scrape *targets* *limits*, which is a different case.

Reply:
I see,  I have add metric to dashboards.libsonnet.

Reply:
![image](https://user-images.githubusercontent.com/7621241/119948310-4877a900-bfcb-11eb-9e59-b9704e72aea5.png)